doc/rst/technotes/gpu.rst

```diff
@@ -178,6 +185,12 @@ section; many of them will be addressed in upcoming editions.
 * Runtime checks such as bounds checks and nil-dereference checks are
   automatically disabled for CHPL_LOCALE_MODEL=gpu.
 
+* For AMD GPUs:
+
+    * Multiple locales are not supported
+
+    * Certain 64-bit math functions are unsupported
```

Comment:
The list of functions can't be reasonably inlined here, can it? Or does it make sense to include a few that we expect to be common? I am fine as-is, too.

Reply:
We could but it's a bit much, there are 19 operations won't link and 1 gives a slightly different result for NVIDIA and AMD. 

Would it be fine if we just linked to the relevant test? Something like this:

```
    * Certain 64-bit math functions are unsupported. To see what does
      and doesn't work see `this test
      <https://github.com/chapel-lang/chapel/blob/main/test/gpu/native/math.chpl>`_
      and note which operations are excluded when `excludeForRocm == true`.
```

Reply:
I see. I am on the fence. Still OK with as-is. I am a little worried that this cross-reference is something we'll need to maintain (ie. is it still going to be relevant if we change that test?)

Reply:
Yeah I feel ambivalent about it as well. If we link to the version off `main` then in some sense it will remain up-to-date as I don't think we'd fail to update that test. Though I suppose I should really link to the 1.30 documentation (https://github.com/chapel-lang/chapel/blob/release/1.30/test/gpu/native/math.chpl)as this tech note really is for that release (I'll do that).

Anyway, I'm guessing at some point we'll just get all of them to work and we can just remove that bullet altogether.

FWIW we link to several other example tests here:
https://chapel-lang.org/docs/technotes/gpu.html#test-examples

Reply:
Yeah good point about other tests. And in general, it makes sense to link to some examples at this stage from this technote. Hopefully, we'll put some inline codes here instead, or maybe write a GPU primer... Not today's work, obviously.

Reply:
Yeah for sure we'll want a primer. Some of these I imagine we'll also promote out into `test/release/examples` but probably not until we think of GPU support as being more "stabilized".

Reply:
This reminded me of https://github.com/Cray/chapel-private/issues/4549, which is only slightly related.